tests/calc/classes/rparams/special/test_max_tl_displacement.py

```diff
@@ -0,0 +1,125 @@
+"""Tests for viperleed.calc.classes.rparams.special.max_tl_displacement."""
+
+__authors__ = (
+    'Florian Kraushofer (@fkraushofer)',
+    )
+__copyright__ = 'Copyright (c) 2019-2025 ViPErLEED developers'
+__created__ = '2025-06-23'
+__license__ = 'GPLv3+'
+
+import pytest
+from pytest_cases import parametrize
+
+from viperleed.calc.classes.rparams.special.max_tl_displacement import (
+    MaxTLAction, MaxTLDisplacement
```

Comment:
```suggestion
    MaxTLAction,
    MaxTLDisplacement,
```

One import per line, so if any is added/removed the commit only involves one line